src/hooks/useAddressGeneration.tsx

```diff
@@ -105,8 +101,6 @@ const useAddressGeneration = () => {
       }))
 
       saveNewAddresses(addresses, { walletName, mnemonic })
-
-      deriveAddressesInGroupsWorker.terminate()
```

Comment:
This had to be removed because after using the `generateAndSaveOneAddressPerGroup` once, it could not be used again.